Uplift/Views/Onboarding/CreateProfileView.swift

```diff
@@ -0,0 +1,190 @@
+//
+//  CreateProfileView.swift
+//  Uplift
+//
+//  Created by Belle Hu on 9/22/24.
+//  Copyright © 2024 Cornell AppDev. All rights reserved.
+//
+
+import PhotosUI
+import SwiftUI
+
+struct CreateProfileView: View {
+
+    // MARK: - Properties
+
+    @EnvironmentObject var mainViewModel: MainView.ViewModel
+    @State private var didCheckTerms: Bool = false
+    @State private var didCheckData: Bool = false
+    @State private var didCheckLocation: Bool = false
+    @State private var profileImage: UIImage?
+    @State private var didShowImagePicker = false
```

Comment:
```suggestion
    @State private var didCheckData: Bool = false
    @State private var didCheckLocation: Bool = false
    @State private var didCheckTerms: Bool = false
    @State private var didShowImagePicker = false
    @State private var profileImage: UIImage?
```
Fix for alphabetical order.